### PR TITLE
modify README including CoC

### PR DESCRIPTION
**Pull request by jbowen93**
Remove the tharsis/ethermint code of conduct and start a fresh README with a link to the celestia code of conduct.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,78 +1,9 @@
-<!--
-parent:
-  order: false
--->
+# Ethermint
 
-<div align="center">
-  <h1> Ethermint </h1>
-</div>
+Celestiaorg fork of github.com/tharsis/ethermint
```
Comment (adlerjohn) [inline README.md:3]: Add scheme for URLs

Comment (jbowen93) [reply README.md:3]: done

Comment (adlerjohn) [reply README.md:3]: Wait, missing scheme on this url 😂

Comment (jbowen93) [reply README.md]: actually done

**Review by adlerjohn — APPROVED** [review]

**Pull request merged**